src/ui/public/partials/saved_object_finder.html

```diff
@@ -22,27 +22,30 @@
     </div>
 
     <div class="kuiBarSection">
-      <p class="kuiText kuiSubduedText">
-        {{ pageFirstItem }}-{{ pageLastItem }} of {{ finder.hitCount }}
-      </p>
-
+      <p class="kuiText kuiSubduedText"
+        i18n-id="common.ui.savedObjectFinder.pageItemsFromHitCountDescription"
+        i18n-default-message="{pageFirstItem}-{pageLastItem} of {hitCount}"
+        i18n-values="{pageFirstItem: pageFirstItem, pageLastItem:pageLastItem, hitCount: finder.hitCount}"
+      ></p>
       <div class="kuiButtonGroup">
         <button
           class="kuiButton kuiButton--primary"
           ng-if="onAddNew"
           ng-click="onAddNew()"
           data-test-subj="addNewSavedObjectLink"
-        >
-          Add new {{ finder.properties.noun }}
-        </button>
+          i18n-id="common.ui.savedObjectFinder.newNounButtonLabel"
```

Comment:
I'd use `item` instead of `noun` word.
```
          i18n-id="common.ui.savedObjectFinder.addNewItemButtonLabel"
          i18n-default-message="Add new {item}"
          i18n-values="{ item: finder.properties.noun }"
```